sdk/src/main/java/com/antithesis/sdk/internal/HandlerFactory.java

```diff
@@ -102,7 +94,6 @@ public void output(final String value) {
                 writer.write("\n");
                 writer.flush();
             } catch (IOException ignored) {
-                // TODO (@shomik) logging
```

Comment:
logging might be abusive here.

Reply:
Missed removing this comment, thank you!